nova/src/provider/zeromorph.rs

```diff
@@ -0,0 +1,108 @@
+use std::marker::PhantomData;
```

Comment:
I could have, but it has its own logging that I didn't want to stomp on. Plus integrating the SRS from SetupAux stuff will add concrete functionality that makes sense here instead of Spartan (as in the pcd/compression code).